crates/re_data_ui/src/component.rs

```diff
@@ -52,6 +52,52 @@ impl DataUi for EntityLatestAtResults {
             UiVerbosity::LimitHeight | UiVerbosity::Full => num_instances,
         };
 
+        // Display log time and additional diagnostic information for static components.
+        if verbosity != UiVerbosity::Small {
+            ui.label(format!(
+                "Timestamp: {}",
+                query
+                    .timeline()
+                    .typ()
+                    .format(self.results.index().0, ctx.app_options.time_zone),
+            ));
+
+            // if the component is static, we display extra diagnostic information
+            if self.results.is_static() {
+                if let Some(histogram) = db
+                    .tree()
+                    .subtree(&self.entity_path)
+                    .and_then(|tree| tree.entity.components.get(&self.component_name))
+                {
+                    if histogram.num_static_messages() > 1 {
+                        ui.label(ctx.re_ui.warning_text(format!(
+                            "Static component value was overridden {} times",
+                            histogram.num_static_messages() - 1,
```

Comment:
I would default to future proofing this, just in case this code is ever copy-pasted without the surround `if` statement...

```suggestion
                            histogram.num_static_messages().saturating_sub(1),